sgid/index.html

```diff
@@ -0,0 +1,110 @@
+---
+title: The SGID
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2020-05-26 13:58:58 -0600
+layout: default
+categories: []
+tags:
+  - sgid
+---
+<div class="grid">
+
+    <div class="grid__col section">
+        <h2>The State Geographic Information Database</h2>
+        <p class="readable">Established by State law in 1991, Utah’s State Geographic Information Database (SGID) provides a centralized collection of hundreds of GIS map data layers developed, aggregated, or acquired by the State. Evolving technologies have allowed us to transform the SGID from a single database to a broad collection of data resources under the SGID umbrella. It contains both AGRC-curated datasets and references to data from a wide variety of state and local agencies. Many resources have both download links and links to web services for direct consumption.</p>
+
+        <p class="readable">There are many pieces, or facets, to the SGID today, with each one addressing specific audiences or technologies. The SGID Index combines all of these along with links to externally-hosted resources to make it easy to find the latest and highest quality GIS data resources no matter where they're hosted.
+
+        <p class="readable">Scroll down to read more about each facet of the SGID:</p>
+            <ul class="readable"> 
+                <li>The curated Open SGID database</li>
+                <li>ArcGIS Online hosted feature services</li>
+                <li>SGID Open Data site</li>
+                <li>Our Raster Discovery web app</li>
```

Comment:
**suggestion:** Change _Our_ to _The_ or _The AGRC_ (to match the bottom bullet)